src/util/contextHelpers.js

```diff
@@ -120,7 +124,9 @@ export const lazyLoadWithDimensions = (Component, options) => {
     }
 
     handleWindowResize() {
-      this.setDimensions();
+      if (this.isElementNearViewport(NEAR_VIEWPORT_MARGIN)) {
```

Comment:
`isElementNearViewport` doesn't seem to take a param, but it uses `NEAR_VIEWPORT_MARGIN` internally.